test/sanity_checks.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
```

Comment:
I think more users are apt to find this out if it's in cri-o itself. we could have a log message come out of `internal/config/node` similarly to how we check other node level knobs. wdyt

Reply:
This one is strictly for CI, so it would fail in case we are not running in a sane env.

Adding a log message to cri-o itself is also on my radar. Added.

Reply:
Not sure if we should make it fatal or not. I'd rather make it fatal as otherwise we'll have to deal with some weird EBUSY bugs for no reason.